gateway/assets/newfunction.html

```diff
@@ -1,40 +1,42 @@
-<md-dialog  aria-label="List dialog" layout="column" flex="70"> 
+<md-dialog aria-label="List dialog" layout="column" flex="70">
     <md-dialog-content class="md-padding">
         <label>Define a function:</label>
         <form name="userForm">
             <div layout-gt-xs="row">
                 <md-input-container class="md-block" flex-gt-sm>
-                <label>Image:</label>
-                <input name="dockerImage" ng-model="item.image" required md-maxlength="200" minlength="4">
```

Comment:
We also took off `required`?

Reply:
I carved out a few minutes to test the PR on my laptop - seems like removing the `required` field may have introduced regression in the UI minimum length / required field status.

I can create a service with no name.. that's not expected behaviour.

Reply:
I've added the attribute back but i'm still able to submit an empty form. When i click `Create` in the new function dialog I get the error `Deployment error: Error response from daemon: rpc error: code = 3 desc = ContainerSpec: image reference must be provided` which shows that a trip to the server is happening.